lua/wikis/ageofempires/GetMatchGroupCopyPaste/wiki.lua

```diff
@@ -66,18 +82,35 @@ end
 
 --subfunction used to generate code for the Map template, depending on the type of opponent
 ---@param mode string
+---@param isFFA boolean
 ---@return string
-function WikiCopyPaste._getMap(mode)
-	local lines = Array.extend(
-		'={{Map',
-		INDENT .. INDENT .. '|map=|winner=',
+function WikiCopyPaste._getMap(mode, isFFA)
```

Comment:
just personal preference but we could avoid `not isFFA`

```lua
	local opponentLines = isFFA and {
		INDENT .. INDENT .. '|date=',
	} or {
		mode == Opponent.team and INDENT .. INDENT .. '|players1=' or nil,
		INDENT .. INDENT .. '|civs1=',
		mode == Opponent.team and INDENT .. INDENT .. '|players2=' or nil,
		INDENT .. INDENT .. '|civs2=',
	}

	local lines = Array.extend(
		'={{Map',
		INDENT .. INDENT .. '|map=' .. (isFFA and '' or '|winner='),
		Array.extractValues(opponentLines),
		INDENT .. '}}'
	)
	return table.concat(lines, '\n')
```

alternatively pass `not isFFA` into this function and name it differently here